docs/plugins/google-cloud.md

```diff
@@ -152,4 +152,4 @@ This option will force Genkit to export telemetry and log data when running in t
 
 ## Test your integration
 
-When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the Firebase AI Monitoring dashboard for an AI-idiomatic view of telemetry.
+When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) for an AI-idiomatic view of telemetry.
```

Comment:
Do we want to show them where this configuration goes? This could be a touch confusing.

Reply:
There is an example of this above under Plugin Configuration, so I am tempted to say no.